packages/falso/src/lib/permission.json

```diff
@@ -0,0 +1,3 @@
+{
+  "data": ["read", "write", "execute", "no permission"]
```

Comment:
Put all the options under data:
```
{
"data": {
  numeric: [0, 1, 2, 4]
  text: ["read", "write", "execute", "no permission"]
  combination:   ['---', '--x', '-w-', '-wx', 'r--', 'r-x', 'rw-', 'rwx']
 }
}
```

Reply:
Good point. My intention was to add only "translatable" content to `data` in order to make translations easy (#1) and to avoid issues with universal/not translatable content, as with `numeric` and `combination` values.

But I don't have a strong opinion on that. If there are no concerns on your side, I would be happy to put everything under `data`. 🙂

Considering #1, we probably need some rules/schema for `*.json` files. 🤔

Reply:
You're doing fine. This separation is important for i18n.